qa-pipelines/generate-vars.rb

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env ruby
```

Comment:
Based on a reading and some discussion, it looks like this file may not actually be simplifying anything. I think the original intent was to feed something into the generate-pipeline file, but that's not happening now

Reply:
Reverted in b2c57c912018ff0f716f9ec90bcbb2448a888802 (or rather, will be when I push it); we might want to `git rebase -i` before merge to drop that one as well as the commit introducing the change.